iphone/TitaniumKit/TitaniumKit/Sources/API/TiUIView.m

```diff
@@ -658,7 +658,11 @@ - (void)setBackgroundImage_:(id)image
   if (backgroundRepeat) {
     [self renderRepeatedBackground:bgImage];
   } else {
+    // To fix TIMOB-28150
+    [CATransaction begin];
+    [CATransaction setValue:(id)kCFBooleanTrue forKey:kCATransactionDisableActions];
```

Comment:
You can use a convenience method here which is a little nicer to read.

```suggestion
    [CATransaction setDisableActions:YES];
```

Reply:
Agreed. Updated!